### ISSUE TITLE
Merge search-result branch

### ISSUE THREAD

**Issue by chrisreu** [hunt-framework/hunt#97]
I think, whereas most of the branches are obsolete or just being created for tests, this particular branch could/should be merged into master before starting any new stuff:

https://github.com/hunt-framework/hunt/tree/new-search-result

**Comment by alexbiehl** [comment]
I fixed the tests for new-search-result branch and merged it into master.
